tests/beaker_tests/lib/utilitylib.rb

```diff
@@ -226,14 +226,25 @@ def format_stepinfo(tests, id, test_str)
 
 # Wrapper for manifest tests
 # Pass code = [0], as an alternative to 'test_idempotence'
-def test_manifest(tests, id)
+# If the user passes in a find_patern, search for it in the output
+def test_manifest(tests, id, find_pattern=nil)
```

Comment:
Nope, don't like this, nuh-uh, not a bit. The harness api's are intended to stay simple, using hash keys for test knobs instead of adding endless parameters. 
Also, doesn't this change duplicate the :show_pattern key?